src/test/ui/rfc-2632-const-trait-impl/const-impl-requires-const-trait.stderr

```diff
@@ -1,14 +1,21 @@
 error: const `impl`s must be for traits marked with `#[const_trait]`
-  --> $DIR/const-impl-requires-const-trait.rs:6:1
+  --> $DIR/const-impl-requires-const-trait.rs:7:1
    |
 LL | impl const A for () {}
    | ^^^^^^^^^^^^^^^^^^^
    |
 note: this trait must be annotated with `#[const_trait]`
-  --> $DIR/const-impl-requires-const-trait.rs:3:1
+  --> $DIR/const-impl-requires-const-trait.rs:4:1
    |
 LL | pub trait A {}
    | ^^^^^^^^^^^
 
-error: aborting due to previous error
+error[E0207]: the effect parameter `constness` is not constrained by the impl trait, self type, or predicates
```

Comment:
This is a fairly broad message. As of now we can't have `constness` on the self type, right? Unless we make `dyn ~const Fn` or `~const fn` a thing.